public_html/wp-content/sunrise.php

```diff
@@ -578,8 +578,8 @@ function get_canonical_year_url( $domain, $path ) {
 			break;
 
 		case "us.wordcamp.$tld":
-			if ( time() <= strtotime( '2022-10-01' ) ) {
-				return "https://us.wordcamp.$tld/2022/";
+			if ( time() <= strtotime( '2023-10-01' ) ) {
```

Comment:
It [looks like](https://us.wordcamp.org/2023/) the dates this year are `August 24-26`

Reply:
yes, I was giving it some padding for post event, and figured that Sept 1 was too soon